concrete/elements/pages/schedule.php

```diff
@@ -41,52 +42,51 @@
+<div class="text-right">
+    <span class="form-text help-block"><?= t('Time Zone: %s', $timezone) ?></span>
 </div>
 
-<?php if ($activeVersionExists || $scheduledVersionExists) {
-    if($liveVersionStatusOnScheduledVersionApproval === 'unapproved') {
-        if ($scheduledVersionExists) {
-            $keepOtherScheduling = t('Remove current scheduled. This version will go live by itself.');
-        } else {
-            $keepOtherScheduling = t('Remove live version in meantime.');
-        }
-    } else {
-        if ($scheduledVersionExists) {
-            $keepOtherScheduling = t('Keep existing scheduling. This version will go live separately.');
-        } else {
-            $keepOtherScheduling = t('Keep live version approved.');
-        }
-    }
-    ?>
-<div class="form-group">
-    <div class="form-check form-switch">
-        <?= $form->checkbox('keepOtherScheduling', 1, false) ?>
-        <?= $form->label('keepOtherScheduling', $keepOtherScheduling) ?>
+<?php if ($activeVersionExists || $scheduledVersionExists) { ?>
+    <div class="form-group">
+        <div id="version-scheduling" class="form-check form-switch">
+            <?= $form->checkbox('keepOtherScheduling', 1, false, ['id' => 'keepOtherScheduling']) ?>
+            <label for="keepOtherScheduling" class="form-check-label">
+                <span class="text-standard"><?= $scheduledVersionExists ? t('At the moment, the existing scheduled version will remain.') : t('At the moment, the existing live version will remain.') ?></span>
+                <span class="text-active d-none"><?= $scheduledVersionExists ? t('At the moment, current scheduled version gets removed.') : t('At the moment, the current live version gets removed.') ?></span>
+            </label>
+            <span class="form-text help-block">
+                <i class="fa fa-info-circle" aria-hidden="true"></i>
+                <span class="info-standard"><?= $scheduledVersionExists ? t('Activate to remove the current scheduled version.') : t('Activate to remove the current live version.') ?></span>
+                <span class="info-active d-none"><?= $scheduledVersionExists ? t('Deactivate to leave current scheduled version.') : t('Deactivate to leave current live version online.') ?></span>
+            </span>
+        </div>
     </div>
-</div>
 <?php } ?>
 
 <div class="dialog-buttons">
-    <button type="submit" name="action" value="schedule"
-            class="btn btn-primary ccm-check-in-schedule">
-        <?=t('Schedule')?>
+    <button type="submit" name="action" value="schedule" class="btn btn-primary">
+        <?= t('Schedule') ?>
     </button>
 </div>
 
-<style type="text/css">
-    div.ui-dialog button.ccm-check-in-schedule {
-        float: right;
-    }
-</style>
+<script>
+$(document).ready(function(){
+    $('#keepOtherScheduling').change(function(){
+        if(this.checked) {
+            $('.text-standard, .info-standard').addClass('d-none');
```

Comment:
This alters every element with `text-standard` or `info-standard` classes, even those we may not want to touch (that is, something else on the page).

What about

```js
$('#version-scheduling .text-standard, #version-scheduling .info-standard').addClass('d-none');
```

that way, we'll only touch the elements that are children of the element with id `version-scheduling`.

Reply:
Thanks for the feedback, @mlocati. I've scoped the CSS to target elements under #version-scheduling, preventing conflicts. Please let me know if there are more changes needed. Thanks!

Reply:
@KnollElias Can you also scope the add and removeClass functions?